cloud_experts_tutorials/cloud-experts-using-alb-and-waf.adoc

```diff
@@ -0,0 +1,443 @@
+
+// Loosely based off EKS instructions here - https://aws.amazon.com/premiumsupport/knowledge-center/eks-alb-ingress-aws-waf/
+
+[id="prerequisites_{context}"]
+== Prerequisites
+
+[NOTE]
+====
+AWS ALBs require a multi-AZ cluster, as well as three public subnets split across three AZs in the same VPC as the cluster. 
+====
+
+* xref:../rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-quickly.adoc#rosa-sts-creating-a-cluster-quickly[A multi-AZ ROSA Classic cluster].
+* You have access to the OpenShift CLI (`oc`).
+* You have access to the AWS CLI (`aws`).
+
+[id="environment-setup_{context}"]
+=== Environment setup
+
+* Prepare the environment variables:
++
+[source,terminal]
+----
+$ export AWS_PAGER=""
+$ export CLUSTER_NAME=$(oc get infrastructure cluster -o=jsonpath="{.status.infrastructureName}"  | sed 's/-[a-z0-9]\{5\}$//')
+$ export REGION=$(oc get infrastructure cluster -o=jsonpath="{.status.platformStatus.aws.region}")
+$ export OIDC_ENDPOINT=$(oc get authentication.config.openshift.io cluster -o jsonpath='{.spec.serviceAccountIssuer}' | sed  's|^https://||')
+$ export AWS_ACCOUNT_ID=$(aws sts get-caller-identity --query Account --output text)
+$ export SCRATCH="/tmp/${CLUSTER_NAME}/alb-waf"
+$ mkdir -p ${SCRATCH}
+$ echo "Cluster: ${CLUSTER_NAME}, Region: ${REGION}, OIDC Endpoint: ${OIDC_ENDPOINT}, AWS Account ID: ${AWS_ACCOUNT_ID}"
+----
+
+[id="aws-vpc-and-subnets_{context}"]
+=== AWS VPC and subnets
+
+[NOTE]
+====
+This section only applies to clusters that were deployed into existing VPCs. If you did not deploy your cluster into an existing VPC, skip this section and proceed to the installation section below.
+====
+
+. Set the below variables to the proper values for your ROSA deployment:
++
+[source,terminal]
+----
+$ export VPC_ID=<vpc-id>
+$ export PUBLIC_SUBNET_IDS=<public-subnets>
+$ export PRIVATE_SUBNET_IDS=<private-subnets>
+----
++
+. Add a tag to your cluster's VPC with the cluster name: 
++
+[source,terminal]
+----
+$ aws ec2 create-tags --resources ${VPC_ID} --tags Key=kubernetes.io/cluster/${CLUSTER_NAME},Value=owned --region ${REGION}
+----
++
+. Add a tag to your public subnets:
++
+[source,terminal]
+----   
+$ aws ec2 create-tags \
+     --resources ${PUBLIC_SUBNET_IDS} \
+     --tags Key=kubernetes.io/role/elb,Value='' \
+     --region ${REGION}
+----
++
+. Add a tag to your private subnets:
++
+[source,terminal]
+----
+$ aws ec2 create-tags \
+     --resources "${PRIVATE_SUBNET_IDS}" \
+     --tags Key=kubernetes.io/role/internal-elb,Value='' \
+     --region ${REGION}
+----
+
+[id="deploy-aws-load-balancer-operator_{context}"]
+== Deploy the AWS Load Balancer Operator
+
+The link:https://github.com/openshift/aws-load-balancer-operator[AWS Load Balancer Operator] is used to used to install, manage and configure an instance of `aws-load-balancer-controller` in a ROSA cluster. To deploy ALBs in ROSA, we need to first deploy the AWS Load Balancer Operator.
```

Comment:
Suggestion: 
Deploy the AWS Load Balancer Operator to enable deployment of ALBs in ROSA.